app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= form_with(model: post, local: true) do |form| %>
+  <% if post.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(post.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+        <% post.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field ">
+    <%= form.label :title %>
+    <%= form.text_field :title, class:"form-control" %>
+  </div>
+
+  <div class="field ">
+    <%= form.label :body %>
+    <%= form.text_area :body, class:"form-control"%>
+  </div>
+
+  <div class="field">
+    <%= form.label :author %>
+    <%= form.text_field :author, class:"form-control" %>
+  </div>
```

Comment:
This field is unnecessary. You can just display the name/username of the author of the post. Asking for the author's name is unnecessary.

Please remove this field. Also, remember to remove the `author` field from the `posts` table.